### PR TITLE
Add doc to create cluster with kubectl

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**

### CODE REVIEW

File: doc/create-cluster-kubectl.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+## Deploy custom resources
+run `make`
+
+```
+% ./bin/clusterctl --kubeconfig=$KUBECONFIG  \
+                   alpha phases \
+                   apply-cluster-api-components \
+                   -p provider-components.yaml
```
Comment (pierre-emmanuelJ) [inline doc/create-cluster-kubectl.md]: or maybe just a `kubectl apply -f provider-components.yaml`

Comment (pierre-emmanuelJ) [reply doc/create-cluster-kubectl.md]: @greut and `clusterctl` in this case is not needed

Comment (greut) [reply doc/create-cluster-kubectl.md]: I'll have to dig the documentation to see what's behind the "apply-cluser-api-components" ;)

Comment (pierre-emmanuelJ) [reply doc/create-cluster-kubectl.md]: Thanks :smile:

Comment (pierre-emmanuelJ) [reply doc/create-cluster-kubectl.md]: I'll try ASAP

**Pull request merged**